api/db-cdr/build.gradle

```diff
@@ -17,7 +17,7 @@ dependencies {
     liquibaseRuntime 'org.liquibase:liquibase-core:3.10.0'
     liquibaseRuntime 'org.liquibase:liquibase-groovy-dsl:3.0.0'
     liquibaseRuntime 'info.picocli:picocli:4.6.1'
-    liquibaseRuntime 'mysql:mysql-connector-java:5.1.37'
```

Comment:
Find solution here: 
https://forum.liquibase.org/t/liquibaseexception-java-lang-classcastexception-class-java-time-localdatetime-cannot-be-cast-to-class-java-lang-string/5059/5